docs/contributing.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to Humanify
```

Comment:
No justification given for this change, please provide information why you are adding this documentation.

Reply:
They're commonly used contributing docs like you see in many repos, should probably be moved to `/CONTRIBUTING.md` instead of `/docs/contributing.md`